File: CSharp/PurgeAndOrganize.cs
using System.Linq;

namespace CSharp
{
    // Given an array of numbers, write a function that returns an array that
    // Has all duplicate elements removed
    // Is sorted from least value to greatest value
    // https://edabit.com/challenge/9Emx6unF8evWiXnJA
    public static class PurgeAndOrganize
    {
        public static double[] UniqueSort(double[] arr) => arr.Distinct().OrderBy(num => num).ToArray();
    }
}

File: python/purge_and_organize.py
"""
Purge and Organize

Given a list of numbers, write a function that returns a list that...
1. Has all duplicate elements removed.
2. Is sorted from least to greatest value.

https://edabit.com/challenge/EZMCpHaNFg2Yfsnxx
"""
from typing import List, Union

NumberList = List[Union[int, float]]


def unique_sort(lst: NumberList) -> NumberList:
    return sorted(set(lst))
